app/views/topics/index.html.erb

```diff
@@ -47,8 +47,8 @@
 
 <div>
   <% if user_signed_in? %>
-    <span class="hidden-xs pull-right"><%= link_to t(:open_new_discussion, default: "Open Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
+    <span class="hidden-xs pull-right"><%= link_to t(:start_discussion, default: "Start Discussion"), new_topic_path, class: 'btn btn-primary' %></span>
```

Comment:
We've renamed `:open_new_discussion` to `:start_discussion` but have the i18n file keys been changed too?